packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -121,21 +131,23 @@ function computePerformanceEntryMetrics(timing: RumPerformanceResourceTiming) {
   }
 }
 
-function computeRequestTracingInfo(request: RequestCompleteEvent) {
-  const hasBeenTraced = request.traceSampled && request.traceId && request.spanId
-  if (!hasBeenTraced) {
-    return undefined
-  }
+function computeRequestTracingInfo(request: RequestCompleteEvent, configuration: RumConfiguration) {
   return {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏What is the reasoning for moving this logic to the callers?

Reply:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** We do some conditional logic to the variables `startClocks` and `correspondingTimingOverrides` in the lines just above the variable `tracingInfo`. IMO it feels nicer to have conditional logic in that one function. 

Happy to revert it if you feel this is unnecessary

Reply:
IMO, this function was encapsulating all the logic around tracing info so it is a bit a shame to split it and duplicate some it. 

To improve the current state, I'd rather limit process functions responsibilities to only run various computations and assemble the results, leaving all the conditional logic to sub parts.
IMO, no need to go there now.

wdyt?

Reply:
Makes sense, I agree having logic coupled and contained to relevant functions makes it a lot easier to read through.

I will revert it and lower the newly added conditional logic down to `computeEntryTracingInfo`